scripts/data_collector/pit/test_pit.py

```diff
@@ -57,9 +63,8 @@ def test_query(self):
 
     def test_no_exist_data(self):
         fields = ["P($$roewa_q)", "P($$yoyni_q)", "$close"]
-        data = D.features(["sh600519", "sh601988"], fields, start_time="2019-01-01", end_time="20190719", freq="day")
+        data = D.features(["sh600519", "sh601988"], fields, start_time="2019-01-01", end_time="2019-01-08", freq="day")
```

Comment:
@Chaoyingz   There is a very special case on 20190713 and 20190718 for MaoTai
I think it would be better to keep the previous example to detect more corner cases. 
![image](https://user-images.githubusercontent.com/465606/158961437-aaa907b8-bcc6-4991-93e6-5661c608dbf0.png)